src/current/v23.2/physical-cluster-replication-monitoring.md

```diff
@@ -101,6 +102,72 @@ We recommend tracking the following metrics:
+    ---+--------------------+-------------+--------------+--------------------+-------------------------------------------------------------------------------------------------------------------+--------------------+-------------------------------+-------------------------------+---------------
+    3  | standbyapplication | replicating | none         | application        | postgresql://{user}:redacted@{node IP}:26257/?options=-ccluster%3Dsystem&sslmode=verify-full&sslrootcert=redacted | 925475002117849089 | 2023-12-15 19:39:18.321455+00 | 2023-12-14 19:16:07.321456+00 |         NULL
+    (1 row)
+    ~~~
+
+    For detail on connecting to the standby cluster, refer to [Set Up Physical Cluster Replication]({% link {{ page.version.version }}/set-up-physical-cluster-replication.md %}#connect-to-the-standby-cluster-system-interface).
+
+1. From the **primary cluster's system interface**, specify a timestamp at or lower than the current `replicated_time` to verify the data:
+
+    {% include_cached copy-clipboard.html %}
+    ~~~ sql
+    SELECT * FROM [SHOW EXPERIMENTAL_FINGERPRINTS FROM VIRTUAL CLUSTER application] AS OF SYSTEM TIME '1702669158089558000.0000000000';
+    ~~~
+    ~~~
+    tenant_name |             end_ts             |     fingerprint
+    ------------+--------------------------------+----------------------
+    application | 1702669158089558000.0000000000 | 2646132238164576487
+    (1 row)
+    ~~~
+
+    For detail on connecting to the primary cluster, refer to [Set Up Physical Cluster Replication]({% link {{ page.version.version }}/set-up-physical-cluster-replication.md %}#connect-to-the-primary-cluster-system-interface).
+
+1. From the **standby cluster's system interface**, specify a timestamp at or lower than the current `replicated_time` to verify the data:
+
+    {% include_cached copy-clipboard.html %}
+    ~~~ sql
+    SELECT * FROM [SHOW EXPERIMENTAL_FINGERPRINTS FROM VIRTUAL CLUSTER standbyapplication] AS OF SYSTEM TIME '1702669158089558000.0000000000';
+    ~~~
+    ~~~
+        tenant_name     |             end_ts             |     fingerprint
+    --------------------+--------------------------------+----------------------
+    standbyapplication  | 1702669158089558000.0000000000 | 2646132238164576487
+    (1 row)
+    ~~~
+
+You can also use `SHOW EXPERIMENTAL_FINGERPRINTS` without defining a timestamp to view the fingerprint for the latest replicated timestamp:
+
+{% include_cached copy-clipboard.html %}
+~~~ sql
+SHOW EXPERIMENTAL_FINGERPRINTS FROM VIRTUAL CLUSTER standbyapplication;
+~~~
+~~~
+     tenant_name     |             end_ts             |     fingerprint
+---------------------+--------------------------------+----------------------
+  standbyapplication | 1702572059741065000.0000000000 | 5089527880146530165
+(1 row)
+~~~
+
```

Comment:
Unfortunately, SHOW EXPERIMENTAL_FINGERPRINTS without a timestamp will take the timestamp as of the read timestamp of the statement's transaction which isn't guaranteed to be consistent.  

We should remove this example. But...I also think we should open an issue to make it work as you've documented here because I think it would be a good feature.

Reply:
Oh! Got it, that makes sense. I have removed, I can open an eng issue for this.